core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -308,7 +308,8 @@ class KafkaConfig private(doLog: Boolean, val props: util.Map[_, _])
   val brokerIdGenerationEnable: Boolean = getBoolean(ServerConfigs.BROKER_ID_GENERATION_ENABLE_CONFIG)
   val maxReservedBrokerId: Int = getInt(ServerConfigs.RESERVED_BROKER_MAX_ID_CONFIG)
   var brokerId: Int = getInt(ServerConfigs.BROKER_ID_CONFIG)
-  val nodeId: Int = getInt(KRaftConfigs.NODE_ID_CONFIG)
+  private[server] var _nodeIf: Int = getInt(KRaftConfigs.NODE_ID_CONFIG)
```

Comment:
Could you apply this change to `brokerId` too?

Reply:
Ah, I miss this, I will add it,